com.unity.multiplayer.mlapi/Runtime/Configuration/NetworkPrefab.cs

```diff
@@ -5,11 +5,21 @@
 namespace MLAPI.Configuration
 {
     /// <summary>
-    /// A class that represents a NetworkPrefab
+    /// Class that represents a NetworkPrefab
     /// </summary>
     [Serializable]
     public class NetworkPrefab
     {
+        /// <summary>
+        /// Asset reference of the network prefab
+        /// </summary>
+        public GameObject Prefab;
+
+        /// <summary>
+        /// Whether or not this is a player prefab
+        /// </summary>
+        public bool IsPlayer;
```

Comment:
Is it worth the breaking change to rename this from PlayerPrefab to IsPlayer?

Reply:
```cs
var variable1 = somePrefabReference.Prefab;
var variable2 = somePrefabReference.PlayerPrefab;

// guess what `variable1` and `variable2` mean and what are the differences
```

vs

```cs
var variable1 = somePrefabReference.Prefab;
var variable2 = somePrefabReference.IsPlayer;

// guess what `variable1` and `variable2` mean and what are the differences
```